src/FileConventions/Library.fs

```diff
@@ -0,0 +1,25 @@
+﻿module FileConventions
+
+open System.Linq
+open System.IO
+open System
+open Mono.Posix
+open Mono.Unix
+open Mono.Unix.Native
+
+let HasCorrectShebang (fileInfo: FileInfo) =
+    let fileText = File.ReadLines(fileInfo.FullName)
+    if fileText.Any() then
+        let firstLine = fileText.First()
+        (
+            firstLine.StartsWith("#!/usr/bin/env fsx") || 

```

Comment:
@realmarv no need for parens here ^ (and same below)

Reply:
Done